src/assets/scripts/client/constants/themes/default/modules/scope.js

```diff
@@ -0,0 +1,137 @@
+import { COLOR } from '../colors';
```

Comment:
just noticed this, we should stay consistent with export/filenames.  In this particular case, I would lean towards singular names. so:
```javascript
import { COLOR } from '../color'; 
```

Its very much a style thing here, but I find it reads better: `COLOR.RED_02` vs `COLORS.RED_02`.  we are only ever dealing with a _single_ value, so it makes sense for it's parent to also be singular.